packages/xrpl/src/models/ledger/AMM.ts

```diff
@@ -76,11 +76,11 @@ export default interface AMM extends BaseLedgerEntry {
    */
   Flags: 0
   /**
-   * This field is missing on AMM but is present on all other account_object returned objects.
+   * This field is missing on AMM but is present on all other returned objects.
    */
   PreviousTxnID: never
   /**
-   * This field is missing on AMM but is present on all other account_object returned objects.
+   * This field is missing on AMM but is present on all other returned objects.
```

Comment:
The previous comment was more factual.

Reply:
This type can be returned with other methods as well, like `ledger_entry`. It's confusing in those contexts.